### ISSUE TITLE
dietpi dashboard Method not allowed

### ISSUE THREAD

**Issue by crodriguezct** [ravenclaw900/DietPi-Dashboard#806]
**Describe the bug**:
When I try to log in to dietpi dashboard through the ip of dietpi:5252 I am loaded with a white website with the text : Method not allowed

**Version of DietPi-Dashboard**: I don't know it, I installed it from the dietpi management panel
**Nightly or stable**: Stable

**Version of DietPi**: 9.3

**Device architecture**: 

**To Reproduce**
After installing dietpi I enter through the ip that has dietpi and the port (x.x.x.x:5252) and a blank website comes up with the text Method not allowed

**Error log**

**Additional context**

**Comment by Joulinar** [comment]
Can you share more details on the SBC used.

**Comment by Joulinar** [comment]
did you probably install the backend only? You would need to install the frontend version. I guess you answered the question regarding the backend with yes. But it would be no 😉

**Comment by Porrumentzio** [comment]
Hi @Joulinar, does Dashboard need a graphical interface (XFCE or so) installed on the server? I expected it to work via external web browser

**Comment by Joulinar** [comment]
> does Dashboard need a graphical interface (XFCE or so) installed on the server
No

> I expected it to work via external web browser
Correct 

But the dashboard has 2 installation variants. A full version and on the other side only the backend. The backend itself only collects data, but does not display it. For this it needs the full version. See our online docs https://dietpi.com/docs/software/system_stats/#__tabbed_1_5

**Comment by Porrumentzio** [comment]
Understood, thanks. Anyway, I don't know how to actually add frontend nodes on the config file. Is that explained somewhere in the docs?

**Comment by Joulinar** [comment]
if you have just one system, install the full version instead of backend.

**Comment by MichaIng** [comment]
Do this, and when this question appears:
```
Would you like to install only the backend, for viewing on another dashboard page?
```
select `No`.

**Comment by mdp18** [comment]
Been having the same issue even after attempting to install the first version. ERR_CONNECTION_REFUSED.

**Comment by msongz** [comment]
> Do a reinstall:
> 
> dietpi-software reinstall 200
> And when this question appears:
> 
> ```
> Would you like to install only the backend, for viewing on another dashboard page?
> ```
> 
> select `No`.

after reinstall and select no, the dashboard still can not access via web

× dietpi-dashboard.service - Web Dashboard (DietPi)
     Loaded: loaded (/etc/systemd/system/dietpi-dashboard.service; enabled; preset: enabled)
     Active: failed (Result: signal) since Sat 2025-05-10 11:22:18 HKT; 12s ago
   Duration: 4ms
    Process: 6379 ExecStart=/opt/dietpi-dashboard/dietpi-dashboard (code=killed, signal=SEGV)
   Main PID: 6379 (code=killed, signal=SEGV)
        CPU: 1ms

May 10 11:22:18 DietPi systemd[1]: Started dietpi-dashboard.service - Web Dashboard (DietPi).
May 10 11:22:18 DietPi systemd[1]: dietpi-dashboard.service: Main process exited, code=killed, status=11/SEGV
May 10 11:22:18 DietPi systemd[1]: dietpi-dashboard.service: Failed with result 'signal'.

**Comment by Joulinar** [comment]
Which SBC you are using?

**Comment by msongz** [comment]
pi5

**Comment by Joulinar** [comment]
You would need to use latest nightly build instead of stable.

**Comment by msongz** [comment]
```
│  - Command: curl -sSfL                                                       │
│ https://nightly.link/ravenclaw900/DietPi-Dashboard/workflows/push-build/main │
│ /dietpi-dashboard-aarch64.zip -o dietpi-dashboard-aarch64.zip                │
│  - Exit code: 22                                                             │
│  - DietPi version: v9.12.1 (MichaIng/master) | HW_MODEL: 5 | HW_ARCH: 3 |    │
│ DISTRO: 7                                                                    │
│  - Error log:                                                                │
│ curl: (22) The requested URL returned error: 404     
```

nightly can't access

**Comment by mdp18** [comment]
I believe this is the latest build: https://github.com/ravenclaw900/DietPi-Dashboard/actions/runs/13870294354 you can get the download link there.

**Comment by MichaIng** [comment]
Yeah, I need to fix the clippy annotations to get the builds back up. Luckily pretty trivial ones.

**Comment by MichaIng** [comment]
I solved the clippy annotations. New nightly builds are available.
